src/KeyValueStore.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018, 2019 Transposit Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Represents a key/value store, such as stash, env, etc.
+ * @export
```

Comment:
@blazarus what are these annotations?

Reply:
I'm not sure. Should I know?

Reply:
ahhh okay. I thought you added them and I just didn't know what tool consumed them. I agree we should remove them for now.